js/pages/Stock/Overview.vue?vue&type=template&id=31afdab6&lang.js

```diff
@@ -0,0 +1,6 @@
+import { openBlock as _openBlock, createBlock as _createBlock } from "vue";
+
+export function render(_ctx, _cache, $props, $setup, $data, $options) 
```

Comment:
This seems like a build artifact.

Reply:
Yes. At some point I completely messed up the rollup config, and that's that. Also, eslint breaks in the vue-rollup config. I have no idea why. Linting in VS Code or with the command line works, so I am not concerned.